torch/ao/pruning/_experimental/pruner/base_structured_sparsifier.py

```diff
@@ -1,17 +1,121 @@
-from typing import Set, Type
+from itertools import chain
 import torch
+import torch.nn.functional as F
 from torch import nn
+from torch.fx import symbolic_trace
 from torch.nn.utils import parametrize
+from typing import Type, Set, Dict, Callable, Tuple, Any
 
 from torch.ao.pruning import BaseSparsifier
 from .parametrization import FakeStructuredSparsity, BiasHook
+from .match_utils import apply_match
+from .convert_functions import (
```

Comment:
in AO codebase `convert` usually means `quantization convert`.  Is there a different name we could use here just to make it obvious that this is not quantization convert?